mappings/maestro_file_centric.json

```diff
@@ -0,0 +1,224 @@
+{
+  "aliases": {
+    "file_centric": {}
+  },
+  "settings": {
+    "index.number_of_shards": 3,
+    "index.max_results_window": 300000,
+    "analysis": {
+      "analyzer": {
+        "autocomplete_analyzer": {
+          "filter": ["lowercase", "edge_ngram"],
+          "tokenizer": "standard"
+        },
+        "autocomplete_prefix": {
+          "filter": ["lowercase", "edge_ngram"],
+          "tokenizer": "keyword"
+        },
+        "lowercase_keyword": {
+          "filter": ["lowercase"],
+          "tokenizer": "keyword"
+        }
+      },
+      "filter": {
+        "edge_ngram": {
+          "max_gram": 20,
+          "min_gram": 1,
+          "side": "front",
+          "type": "edge_ngram"
+        }
+      }
+    }
+  },
+  "mappings": {
+    "dynamic": false,
+    "date_detection": false,
+    "properties": {
+      "file_autocomplete": {
+        "fields": {
+          "analyzed": {
+            "analyzer": "autocomplete_analyzer",
+            "search_analyzer": "lowercase_keyword",
+            "type": "text"
+          },
+          "lowercase": {
+            "analyzer": "lowercase_keyword",
+            "type": "text"
+          },
+          "prefix": {
+            "analyzer": "autocomplete_prefix",
+            "search_analyzer": "lowercase_keyword",
+            "type": "text"
+          }
+        },
+        "type": "keyword"
+      },
+      "study_id": {
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "object_id": {
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "file_type": {
+        "type": "keyword"
+      },
+      "data_type": {
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "file_access": {
+        "type": "keyword"
+      },
+      "analysis": {
+        "properties": {
+          "analysis_id": {
+            "type": "keyword",
+            "copy_to": ["file_autocomplete"]
+          },
+          "analysis_type": {
+            "type": "keyword"
+          },
+          "analysis_version": {
+            "type": "integer"
+          },
+          "workflow": {
+            "properties": {
+              "workflow_name": {
```

Comment:
Here is the `workflow` section from a real SONG analysis, should the fields name be the same here? like `name` instead of `workflow_name` etc.
```
"workflow": {
  "name": "dna-seq-alignment",
  "inputs": [
    {
      "analysis_type": "sequencing_experiment",
      "input_analysis_id": "bdbd3178-b38e-492b-bd31-78b38e892bb8"
    }
  ],
  "run_id": "wes-6098c99381454683abcce4b99a85bb5c",
  "version": "1.0.5"
}
```

Reply:
yes there should be field names parity to be indexed correctly because maestro copies fields as is, if a rename is desired, it should be done in song meta schema

Reply:
changed